### PR TITLE
chore(package): change npm scope to @ibmdotcom

### PR DESCRIPTION
**Pull request by kennylam**
### Description

This changes the NPM scope to use `@ibmdotcom`.

### Changelog

**Changed**

- change scope to `@ibmdotcom`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "beacon-for-ibm-dotcom",
+  "name": "@ibmdotcom/beacon-for-ibm-dotcom",
```
Comment (jeffchew) [inline package.json:2]: @kennylam should we just call it `@ibmdotcom/beacon`?

Comment (kennylam) [reply package.json:2]: Sure, we can. I'll make the change

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,2 +1,2 @@
 # Beacon for IBM.com team
-* @IBM/beacon-for-ibm-com-members
+* @IBM/beacon-members
```
Comment (kennylam) [inline .github/CODEOWNERS:2]: @jeffchew Not sure if this is the correct change. Is it `<repo-name>-members`?

Comment (jeffchew) [reply .github/CODEOWNERS:2]: we would have to change the name of the team too, let's leave this one alone for now

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -28,6 +28,6 @@ fit within the scope of any of the existing doc fix projects.
 
 ## Submission Guidelines
 
-Read more about our [submission guidelines](https://github.com/IBM/beacon-for-ibm-dotcom/blob/main/docs/submission-guidelines.md).
+Read more about our [submission guidelines](https://github.com/IBM/beacon/blob/main/docs/submission-guidelines.md).
```
Comment (jeffchew) [inline .github/CONTRIBUTING.md:31]: The repo name won't change though, right?

Comment (kennylam) [reply .github/CONTRIBUTING.md:31]: I figured we'd keep it consistent, npm-style. You want the repo name to stay the same?

**Review by jeffchew — APPROVED** [review]
LGTM!

**Pull request merged**